### PR TITLE
Remove hc 9

### PR DESCRIPTION
**Pull request by kapapp**
Genre and difficulty in playlists are no longer hardcoded values, but come from the playlists themselves. Playlists are created with null difficulty, which is then overwritten by playlist value. Removed some hardcoded values & added some image descriptors to xml

### CODE REVIEW

File: app/src/main/res/layout/playlist_expanded_cardview.xml [hunk 1]
```diff
@@ -104,7 +107,7 @@
                     app:layout_constraintStart_toEndOf="@+id/coverGuideline"
                     app:layout_constraintTop_toTopOf="parent"
                     app:layout_constraintVertical_bias="1.0"
-                    tools:text="LONG PLAYLIST HAHAH" />
+                    tools:text="LONG PLAYLIST HAHA" />
```
Comment (marluxiaboss) [inline app/src/main/res/layout/playlist_expanded_cardview.xml:110]: That's still an hardcoded string here.

**Review by marluxiaboss — APPROVED** [review]
Good changes. But you should also check for hardcoded strings in XMLs

**Review by CHOCOLVT — APPROVED** [review]
great PR :
updates are good and not too numerous

**Review by ghemDD — COMMENTED** [review]
Looks good 
One improvement would be to adapt the color of the tag to the difficulty level (Hard - Red, Medium - Orange, Easy - Green)

**Pull request merged**